### ISSUE TITLE
they updated the code

### ISSUE THREAD

**Issue by cjiefeng** [RussellOng95/CS3243-Group5#2]
![image](https://user-images.githubusercontent.com/32985333/55253218-b9a65f00-528f-11e9-9513-29e71b6f2ef8.png)

**Comment by RussellOng95** [comment]
yep i think ours is the latest O_O!! actually idk how check but i pip is the same ver o:
